components/external_server/src/database/sessions.py

```diff
@@ -27,6 +27,6 @@ def delete(database: Database, session_id: SessionId) -> None:
     database.sessions.delete_one(dict(session_id=session_id))
 
 
-def get(database, session_id) -> SessionId:
+def get(database, session_id):
```

Comment:
?

Reply:
The function isn't returning a SessionId but a session from the database, so the `-> SessionId` is definitely wrong. @emdv88 created a Session model class in src/model/session.py, but this function is not returning a instance of that Session class so I'm not really sure how to fix this at the moment.